demo/demo-edge/edge-service/src/main/java/io/servicecomb/demo/edge/service/EdgeDispatcher.java

```diff
@@ -41,10 +44,13 @@ public void init(Router router) {
   protected void onRequest(RoutingContext context) {
     Map<String, String> pathParams = context.pathParams();
     String microserviceName = pathParams.get("param0");
-    String path = "/" + pathParams.get("param1");
+    String pathVersion = pathParams.get("param1");
+    String path = context.request().path().substring(4);
```

Comment:
How can we tell the path by removing the first 4 characters?

Reply:
this is a demo
and the path rule is :/api/ms/version/......